src/style.css

```
@@ -0,0 +1,81 @@
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.block {
+  flex-basis: 100%;
+  height: 300px;
+  display: flex;
+  font-family: Arial, sans-serif;
+  font-size: 100px;
+  color: white;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+}
+
+.block--1 {
+  background-color: #f00;
+  order: 1;
+}
+
+.block--2 {
+  background-color: #c00;
+  order: 2;
+}
+
+.block--3 {
+  background-color: #900;
+  order: 3;
+}
+
+.block--4 {
+  background-color: #600;
+  order: 4;
+}
+
+.block--5 {
+  background-color: #300;
+  order: 5;
+}
+
+.block--6 {
+  background-color: black;
+  order: 6;
+}
+
+@media (min-width: 900px) {
+
+  .block {
+    flex-basis: 33.3%;
```

```suggestion
    flex-basis: 33.33%;
```
Сотые тоже указывай, сейчас видна белая полоска справа на экране